src/modules/crud.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Remember to always try to avoid using linter-disable when possible. For example, this linter error is because you can't import and export between two files, one can import from a separate file and export to another, but they can't cycle between one another. 
- To solve this, you can remove the module and paste your code into the primary JS file or the file you believe it to be adequate to.